camille/source/windiris.py

```diff
@@ -47,33 +51,87 @@ def _sqlite(start_date,
         'Roll': 'roll',
     }, inplace=True)
     df.index.name = 'time'
+    df.index = df.index.tz_convert(tzinfo)
     df.pitch = df.pitch.apply(radians)
     df.roll = df.roll.apply(radians)
 
     return df
 
 
-def windiris(root):
+def windiris(root, tzinfo=pytz.utc):
+    """
+    Creates a function that can be used to read windiris data from
+    specified root directory
+
+    Parameters
+    ----------
+    root : str
+        Path to windiris data root directory
+    tzinfo : datetime.tzinfo
+        Timezone format for returned timeseries
+    Returns
+    -------
+    function (str, datetime.datetime, datetime.datetime, str, str, str)
+        Function for reading windiris time series from db file
+
+        installation : str
+            Installation directory inside root, part of the file name
```

Comment:
Installation is a name specifying which windmill to collect data from. The fact that it is also a folder and table name is not something the user of this function should be concerned with.